distribution/docs/src/main/resources/_contents/installing-contents.adoc

```diff
@@ -1,40 +1,40 @@
 
-=== Installing ${branding}
-
 [IMPORTANT]
 ====
 Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
 ====
 
-==== Prerequisites
-
-===== Directory Permissions
-
-Restrict access to sensitive files by ensuring that the only users with access privileges are administrators.
+=== Installation Prerequisites
 
-====== Directory Permissions on Windows
+These are the system/environment requirements to configure _prior_ to an installation.
 
-. Right-click on the file or directory noted below then select *Full Control → Administrators → System*.
-. Click *Properties → Security → Advanced* and select Creator Owner for `INSTALLATION_HOME` (e.g., `C:\ddf`).
-. Restrict access to sensitive files by ensuring that only *System* and *Administrators* have Full Control to the below files by right-clicking on the file or directory below then selecting *Properties → Security → Advanced*
-. Delete any other groups or users listed with access to `INSTALLATION_HOME/etc` and `INSTALLATION_HOME/deploy`.
+==== Java Requirements
 
 * Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
 ** The recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or later.
```

Comment:
I think it's important to note that your java version absolutely cannot contain non-number values. For example, if you do `java -version` and see `java version "1.8.0_60"` you are fine, but if you see something like `java version "1.8.0_60-ubuntu"`, karaf will throw exceptions and fail.